audit-appendix.html

```diff
@@ -98,6 +102,8 @@ <h3>Ingest</h3>
       <h3>Replication</h3>
       <p>The Replication event represents the distribution of a filegroup or a file to a single location within a DDP.</p>
 
+      <p>It is up to the discretion of the DDP if it wants to provide a Replication event for each copy of the filegroup made or
```

Comment:
Would like to get some feedback on this. I wanted to clarify that DDPs operate differently, and that Replication events can be either for all copies or for the primary copy. Generally we do for all so we could also modify this to just say a Replication event should exist for each copy.